apps/flowguard/templates/flow/monitor.html

```diff
@@ -1,94 +1,101 @@
+                });
             });
-        });
-    </script>
+        </script>
 
-    {% for channel in channels %}
-        <div class="col-md-6 my2" data-channel-pk="{{ channel.pk }}">
-            <div class="card">
-                <div class="card-body">
-                    <div class="row">
-                        <div class="col">
-                            <h1 class="data-channel-name">{{ channel.name }}</h1>
+        <div class="row">
+        {% for channel in channels %}
+            <div class="col-md-6 my-2" data-channel-pk="{{ channel.pk }}">
+                <div class="card">
+                    <div class="card-body">
+                        <div class="row">
+                            <div class="col">
+                                <h1 class="data-channel-name">{{ channel.name }}</h1>
+                            </div>
+                            <div class="col">
+                                <p class="text-right">
+                                    <time class="data-time">...</time>
+                                    <span class="data-live livedot"></span>
+                                </p>
+                            </div>
                         </div>
-                        <div class="col">
-                            <p class="text-right">
-                                <time class="data-time">...</time>
-                                <span class="data-live livedot"></span>
-                            </p>
-                        </div>
-                    </div>
-                    <h1 class="display-3">
-                        <span class="data-value">{{ channel.value.value }}</span>
-                        <small>ℓ</small>
-                    </h1>
-                    <div class="row">
-                        <div class="col">
-                            <h3 class="text-success">
-                                +
-                                <span class="data-diff">...</span>
-                                <small>ℓ</small>
-                                <button class="btn btn-sm btn-warning" type="button" onclick="reset(event, {{ channel.pk }})">
-                                    Reset
-                                </button>
-                            </h3>
+                        <h1 class="display-3">
+                            <span class="data-value">{{ channel.value.value }}</span>
+                            <small>ℓ</small>
+                        </h1>
+                        <div class="row">
+                            <div class="col">
+                                <h3 class="text-success">
+                                    +
+                                    <span class="data-diff">...</span>
+                                    <small>ℓ</small>
+                                    <button class="btn btn-sm btn-warning" type="button" onclick="reset(event, {{ channel.pk }})">
+                                        Reset
```

Comment:
Maybe add translation tags

Reply:
Good point, although I've kind of neglected that in general. I think it's best to make it a separate issue.